ts/components/wallet/FavoriteMethodSwitch.tsx

```diff
@@ -0,0 +1,57 @@
+import { View } from "native-base";
+import * as React from "react";
+import { ActivityIndicator, StyleSheet } from "react-native";
+import { H4 } from "../core/typography/H4";
+import I18n from "../../i18n";
+import { H5 } from "../core/typography/H5";
+import { IOStyles } from "../core/variables/IOStyles";
+import Switch from "../ui/Switch";
+
+type Props = {
+  isLoading?: boolean;
+  onValueChange?: (value: boolean) => void;
+  switchValue: boolean;
+};
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  },
+  left: {
+    ...IOStyles.flex,
+    paddingRight: 8
+  },
+  right: {
+    justifyContent: "flex-start"
```

Comment:
What do you think to add a fixed dimension to this container style? 
This avoid the movement of the text when the loading indicator is shown that seems like a glitch.
```suggestion
    justifyContent: "flex-start",
    minWidth: 50
```

https://user-images.githubusercontent.com/11773070/112159601-a5309f00-8be9-11eb-958d-0c698d04c64c.mov

Reply:
Absolutely better
changed in https://github.com/pagopa/io-app/pull/2930/commits/b3427893c23b94578344dbd9c858f0be9fa8e6c3
I also updated the video in the PR description